improver/nbhood/__init__.py

```diff
@@ -30,8 +30,12 @@
 # POSSIBILITY OF SUCH DAMAGE.
 """init for nbhood"""
 
+from typing import List, Optional, Tuple, Union
 
-def radius_by_lead_time(radii, lead_times):
+
+def radius_by_lead_time(
+    radii: List[str], lead_times: List[str]
+) -> Tuple[Union[float, List[float], Optional[List[int]]]]:
```

Comment:
It looks as though ```lead_times``` should be optional with a default value of None.

Reply:
Fixed